modeline/disk/disk.asd

```diff
@@ -1,11 +1,12 @@
 ;;;; disk.asd
 
-(asdf:defsystem #:disk
+(asdf:defsystem :disk
```

Comment:
Please use strings for system names instead of keywords. 

Sidenote: Uninterned symbols are preferred over keywords in defpackage forms

Reply:
Sure!